.travis.yml

```diff
@@ -23,12 +24,14 @@ matrix:
     exclude:
       - python: "2.7"
         env: DJANGO=2.0
+      - python: "2.7"
+        env: DJANGO=2.1
       - python: "2.7"
         env: DJANGO=master
       - python: "3.4"
         env: DJANGO=master
-      - python: "3.5"
-        env: DJANGO=master
```

Comment:
Django 2.2 officially supports Python 3.5. (see [docs](https://docs.djangoproject.com/en/2.1/faq/install/#what-python-version-can-i-use-with-django)).

Reply:
The reason that was excluded was that I only really care about testing the master channel in the latest python version.  Even if python 3.5 is supported, I'd rather not have to wait for too many build jobs to finish.